examples/siro_sandbox/app_states/app_state_fetch.py

```diff
@@ -285,12 +622,49 @@ def sim_update(self, dt, post_sim_update_dict):
             self._sandbox_service.end_episode()
             post_sim_update_dict["application_exit"] = True
 
-        if self._sandbox_service.gui_input.get_key_down(GuiInput.KeyNS.M):
-            self._sandbox_service.end_episode(do_reset=True)
+        # use 1-5 keys to select certain episodes corresponding to our 5 scenes
+        num_fetch_scenes = 5
+        # hand-picked episodes from hitl_vr_sample_episodes.json.gz
+        episode_id_by_scene_index = ["0", "5", "10", "15", "20"]
+        for scene_idx in range(num_fetch_scenes):
+            key = GuiInput.KeyNS(GuiInput.KeyNS.ONE.value + scene_idx)
+            if self._sandbox_service.gui_input.get_key_down(key):
+                self._sandbox_service.episode_helper.set_next_episode_by_id(
+                    episode_id_by_scene_index[scene_idx]
+                )
+                self._sandbox_service.end_episode(do_reset=True)
```

Comment:
I don't know what is in `hitl_vr_sample_episodes.json.gz`, but in general this kind of application-specific code in an AppState doesn't bother me. AppStateFetch (or whatever we end up renaming this AppState to) shouldn't aim to be generic; nobody should inherit from it or otherwise build on top of it. It is example code for others to customize in-place or copy-paste from.

Reply:
`hitl_vr_sample_episodes.json.gz` was the set of episodes that we used for the P0 project (with the 5 scenes).